cpp/ql/lib/semmle/code/cpp/ir/implementation/raw/internal/TranslatedExpr.qll

```diff
@@ -3449,7 +3471,13 @@ class TranslatedVarArg extends TranslatedNonConstantExpr {
     result = this.getInstruction(VarArgsVAListStoreTag())
     or
     tag = VarArgsVAListStoreTag() and
-    result = this.getParent().getChildSuccessor(this, kind)
+    (
+      result = this.getParent().getChildSuccessor(this, kind)
+      or
+      // All store instructions could throw an SEH exception
+      result = this.getParent().getExceptionSuccessorInstruction(any(GotoEdge e), true) and
+      kind.(ExceptionEdge).isSEH()
```

Comment:
I'm not sure var args operations can ever throw an SEH exception. So I think we should either:
- Confirm that they can and keep this new edge, or
- Remove these edges.

I'll let you decide on which one you want to pursue :)

Reply:
Removed because I'm not sure what it means to "Translate a var arg" regardless, in the IR semantics.